data/steam

```diff
@@ -3,11 +3,13 @@ s.team
 steam-chat.com
 steamcommunity.com
 steamcontent.com
+steamdeck.com
 steamgames.com
 steampowered.com
 steamstatic.com
 steamusercontent.com
 underlords.com
+valve.net
```

Comment:
can you provide some info about sites under the `valve.net` domain?

Reply:
Sorry, this domain was expired. I will remove that immediately.